example-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -78,6 +82,10 @@ export default function CollectCardPaymentScreen() {
   const [enableCustomerCancellation, setEnableCustomerCancellation] =
     useState(false);
   const [tipEligibleAmount, setTipEligibleAmount] = useState('');
+  const paymentMethodTypes = PAYMENT_METHOD_TYPES;
```

Comment:
It's even trickier now if we use string as the type. In this file we have `paymentMethods` and `paymentMethodTypes` without easy understanding which is which and the responsibility.

Reply:
renamed the `paymentMethods` to `resolvedPaymentMethodTypes`